[2023-04-27T14:43:00.768+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube.fetchComments manual__2023-04-27T14:42:59.147631+00:00 [queued]>
[2023-04-27T14:43:00.774+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube.fetchComments manual__2023-04-27T14:42:59.147631+00:00 [queued]>
[2023-04-27T14:43:00.774+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-27T14:43:00.774+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-27T14:43:00.774+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-27T14:43:00.783+0000] {taskinstance.py:1309} INFO - Executing <Task(_PythonDecoratedOperator): fetchComments> on 2023-04-27 14:42:59.147631+00:00
[2023-04-27T14:43:00.788+0000] {standard_task_runner.py:55} INFO - Started process 3440 to run task
[2023-04-27T14:43:00.790+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'youtube', 'fetchComments', 'manual__2023-04-27T14:42:59.147631+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/youtube.py', '--cfg-path', '/tmp/tmpwi6e2crd']
[2023-04-27T14:43:00.792+0000] {standard_task_runner.py:83} INFO - Job 16: Subtask fetchComments
[2023-04-27T14:43:00.836+0000] {task_command.py:389} INFO - Running <TaskInstance: youtube.fetchComments manual__2023-04-27T14:42:59.147631+00:00 [running]> on host 9e6433510ba5
[2023-04-27T14:43:00.920+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=youtube
AIRFLOW_CTX_TASK_ID=fetchComments
AIRFLOW_CTX_EXECUTION_DATE=2023-04-27T14:42:59.147631+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-27T14:42:59.147631+00:00
[2023-04-27T14:43:00.942+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 217, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/youtube.py", line 19, in fetchComments
    client = GoogleDiscoveryApiHook(api_service_name='youtube', api_version='v3')
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/discovery_api.py", line 67, in __init__
    impersonation_chain=impersonation_chain,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 234, in __init__
    self.extras: dict = self.get_connection(self.gcp_conn_id).extra_dejson
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/connection.py", line 435, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `google_cloud_default` isn't defined
[2023-04-27T14:43:00.950+0000] {taskinstance.py:1332} INFO - Marking task as FAILED. dag_id=youtube, task_id=fetchComments, execution_date=20230427T144259, start_date=20230427T144300, end_date=20230427T144300
[2023-04-27T14:43:00.967+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 16 for task fetchComments (The conn_id `google_cloud_default` isn't defined; 3440)
[2023-04-27T14:43:01.001+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-04-27T14:43:01.017+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
